.gitignore

```
@@ -0,0 +1,11 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.venv/
+.pytest_cache/
+**__pycache__/
+*.pyc
+**db.sqlite3
```

This pattern is a bit too specific as it only ignores a file named exactly `db.sqlite3`. The project's checklist advises against pushing any database files (e.g., with extensions `.sqlite`, `.db3`). A more robust approach is to ignore files by their extension, which prevents accidental commits of any database file, regardless of its name.

```suggestion
**/*.sqlite3
```